contracts/assemblyscript/nep21-basic/README.md

```diff
@@ -24,11 +24,11 @@ The full implementation in Rust can be found there: https://github.com/nearproto
 - Minting and burning
 - Precision, naming and short token name.
 
-# Notable limitations of this implementation
+# Notable limitations of this implementation [this section not needed, given Drawbacks section above]
 
-- Anyone can mint tokens (!!)
+- Anyone can mint tokens (!!) [in Drawbacks above it says that minting is not supported; which is it?]
```

Comment:
this was from my copy-pasta, right now the extended interface allows anyone to mint tokens

Reply:
I left the README as-is for now; I'm happy to clean it up after we meet synchronously, unless you beat me to it